src/campsite-info/components/camp-rating/camp-rating.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Rating } from '@mui/material';
+import StarIcon from '@mui/icons-material/Star'
+import { ratingStyles, tooltipStyles } from './camp-rating.styles';
+
+const tooltipLabels: { [index: string]: string} = {
```

Comment:
Two comments on this typing:
1. In general, if you have a subset of strings that are allowed for something it's better to restrict the type to just those strings. Two ways to do that: union types or enums.
2. When making object types, the `Record` utility type is extremely nice. It works like this: `Record<key type, value type>`